### PR TITLE
Eslint

### PR DESCRIPTION
**Pull request by wonmaungthein**
Added Eslint and set up for both client and serverside to be able to run easily. Updated Readme file too.

### CODE REVIEW

**Comment by wonmaungthein** [comment]
[Chore / Add eslint to both server and client](https://trello.com/c/9XmmnEkA/38-chore-add-eslint-to-both-server-and-client)

**Comment by wonmaungthein** [comment]
[Chore / Add eslint to both server and client](https://trello.com/c/9XmmnEkA/38-chore-add-eslint-to-both-server-and-client)

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # eribognal
 Scotland Graduation Project - Food Mapping Glasgow
 
-## How to run server locally
-To run the server with auto reload, then run the command “npm run dev”.
\ No newline at end of file
+## How to run the App locally
+`npm install` from root directory will install both client and server
+`npm start` will run both server and client.
```
Comment (kabaros) [inline README.md]: `np start` add that it is from root directory too

File: server/package.json [hunk 2]
```diff
@@ -17,5 +17,8 @@
     "morgan": "~1.9.0",
     "nodemon": "^1.12.1",
     "serve-favicon": "~2.4.5"
+  },
```
Comment (kabaros) [inline server/package.json]: can you remove eslint - just run `npm uninstall --save eslint` (and double check it was removed from package.json - if it's not, remove manually)

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (kabaros) [inline .eslintrc.js]: delete this file please (.eslintrc.js)

File: server/package.json (hunk 2 above)
Comment (wonmaungthein) [reply server/package.json]: I have done it.. Thanks

**Pull request closed without merging**